coriolisclient/cli/utils.py

```diff
@@ -114,3 +115,14 @@ def format_json_for_object_property(obj, prop_name):
         prop = prop.to_dict()
 
     return json.dumps(prop, indent=2)
+
+
+def validate_uuid(uuid_string):
+    try:
+        val = UUID(uuid_string, version=4)
+    except ValueError:
+        # If it's a value error, then the string
+        # is not a valid hex code for a UUID.
+        return False
+
+    return str(val) == uuid_string
```

Comment:
This is bad, as `UUID.__str__` always prints the UUID representation with lowercase hex chars, while there's no reason that the given `uuid_str` can't be uppercase.
Considering my other comments, by this point you can safely just `return True`